examples/mercure-asyncapi.yml

```diff
@@ -10,30 +10,29 @@ servers:
     pathname: /.well-known/mercure
     protocol: mercure
 channels:
-  'https://example.com/books/{id}':
+  books:
     address: 'https://example.com/books/{id}'
     messages:
-      publish.message:
-        $ref: '#/components/messages/book'
-      subscribe.message:
+      book:
         $ref: '#/components/messages/book'
     description: >-
       Every time a resource of type `http://schema.org/Book` is created or
       modified, a JSON-LD representation of the new version of this resource
       must be pushed in this Mercure topic.
     parameters:
-      id: {}
+      id:
```

Comment:
this one was converted correct, as type was integer and no more info, even description provided, so result is basically an empty id parameter. But yeah, for best practice better share that description is a minimum